the_big_bang.ipynb

```diff
@@ -0,0 +1,295 @@
+{
```

Comment:
<p>It's kind of tricky, because the solution now implies that you have to use one object, and I expected some candidates to fail to solve it with one object... WDYT?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SygniaLabs/backend-dev-task/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>